contexts/odrl.js

```diff
@@ -0,0 +1,203 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+module.exports = {
+  '@context': {
```

Comment:
Do we need to embed `odrl` context here, since `vc-js` also embeds it?

Reply:
The document loader in this module overrides the default document loader in vc-js.  https://github.com/digitalbazaar/vc-js-cli/blob/initial/lib/api.js#L53

The document loader here will go out to the web to fetch contexts, keys etc. (this was a design requirement): https://github.com/digitalbazaar/vc-js-cli/blob/initial/lib/document-loader.js

The document loader in vc-js does not allow for fetching arbitrary documents from the web: https://github.com/digitalbazaar/vc-js/blob/node-12/lib/documentLoader.js#L16